packages/flutter_adaptive_scaffold/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.1.11+1
```

Comment:
I missed that the version change itself was lost from the PR, which is part of why CI is failing.